lib/models/project.js

```diff
@@ -349,7 +349,8 @@ class Project {
       path.join(internalMiddlewarePath, 'testem-url-rewriter'),
       path.join(internalMiddlewarePath, 'tests-server'),
       path.join(internalMiddlewarePath, 'history-support'),
-      path.join(internalMiddlewarePath, 'serve-files'),
+      path.join(internalMiddlewarePath, 'ember-cli:broccoli:watcher'),
+      path.join(internalMiddlewarePath, 'ember-cli:broccoli:serve-files'),
```

Comment:
Fixed with replacing it with `-` and dropping `ember-cli` since the name was turning too long.